### ISSUE TITLE
Ingest reference information into the Twinkles database?

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTDESC/Twinkles#255]
In our meeting, I heard mention of querying back to FatBoy to get the reference objects and their light curves. Could/should we ingest this information into the Twinkles database at the point when we make the instance catalogs for PhoSim? Thoughts, @rbiswas4 @danielsf @jbkalmbach @jchiang87 ?

**Comment by rbiswas4** [comment]
> I heard mention of querying back to FatBoy to get the reference objects and their light curves.
Yes, and I kind of forgot to mention, if necessary this could be in a file too.

> Could/should we ingest this information into the Twinkles database at the point when we make the instance catalogs for PhoSim?

We could, but is the advantage you are thinking of that we have subcatalogs of the reference catalog?

**Comment by drphilmarshall** [comment]
Yes, and it would make the analysis faster by having everything right there
in the same database - the Monitor would not have to searching around the
UW campus for the old reference objects, and then remake the light curves.

**Comment by jchiang87** [comment]
I'm open to ingesting the reference object info into a local database, e.g., the `DESC_Twinkles_Level_2` db at NERSC, but I'd like to see what the code looks like to get that info from fatboy and how complicated it would be (e.g., schema-wise, etc.) to do the local ingest.   If we get things set up so that fatboy doesn't require special access (i.e., whitelist the NERSC nodes), and we can write code to abstract that querying, we may not need to ingest the desired subset of that data locally.

**Comment by danielsf** [comment]
Cori has already been whitelisted and is able to access fatboy.

It would not be hard to add Edison, too

**Comment by rbiswas4** [comment]
probably for @jchiang87
